test/jdk/java/awt/FullScreen/UninitializedDisplayModeChangeTest/DisplayModeChanger.java

```diff
@@ -64,6 +66,7 @@ public void run() {
                         if (f != null) {
                             f.dispose();
                         }
+                        gd.setDisplayMode(currentDM);
```

Comment:
Should not this be already reset by the gd.setFullScreenWindow(null)

Reply:
I think you may have missed the point as to why this test is unstable.
Look at all the code running on the EDT which constructs the UI that should be either not run there or should be in separate calls.

Reply:
>I dont see it does..It sets the bounds as I see, it does not reset the display mode..
>but say, even though it does, will it be a problem as this will be a no-op (its like double free)

It should be done by this part:
https://github.com/openjdk/jdk/blob/25a4e26320340cdda082cd45639e73b137ce45a2/src/java.desktop/macosx/classes/sun/awt/CGraphicsDevice.java#L238